### PR TITLE
Add in shipment approval information

### PR DESCRIPTION
**Pull request by ryannhg**
Document extra shipment approval information.

Approved shipments will have date and number.

Rejected shipments will have a reason.

If account doesn't support shipment approval information, the shipments list will be empty.

### CODE REVIEW

File: brokers/webhook.md [hunk 1]
```diff
@@ -46,7 +46,23 @@ choose which payloads you want to use and ignore the rest.
       "note": "Put load id on check.",
       "load": {
         "external_id": "load-external-id", // YOUR internal id
-        "shipments": [],
+        "shipments": [
+          {
+            "status": "approved",
+            "details": {
+              "id": "shipment-1",
```
Comment (r38y) [inline brokers/webhook.md:53]: We need it called `external_id` and at the top-level of the hash to match the convention we have in other places.

Comment (r38y) [reply brokers/webhook.md:53]: For example, this in documents:

```
"shipments": [
          {"external_id": "shipment-external-id"}
        ]
```

File: brokers/webhook.md [hunk 2]
```diff
@@ -46,7 +46,23 @@ choose which payloads you want to use and ignore the rest.
       "note": "Put load id on check.",
       "load": {
         "external_id": "load-external-id", // YOUR internal id
-        "shipments": [],
+        "shipments": [
+          {
+            "status": "approved",
+            "details": {
+              "external_id": "shipment-1",
```
Comment (r38y) [inline brokers/webhook.md:53]: I know you hate me, but can you put it at the top level, where status is?

**Comment by kaleposhobios** [comment]
This is done. The code exists but it's not applicable to most customers.

**Pull request closed without merging**